crates/xwt-wtransport/Cargo.toml

```diff
@@ -10,6 +10,8 @@ An implementation of the xwt that runs natively via wtransport crate.
 repository = "https://github.com/MOZGIII/xwt"
 
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
+pin-project-lite = "0.2"
```

Comment:
These are grouped: path deps in first group, crates-io deps in the second group.